doc/developers/advanced_installation.rst

```diff
@@ -112,16 +126,16 @@ builds the extension in place and creates a link to the development directory
 
 .. note::
 
-    If you decide to do an editable install you have to rerun::
+    You will have to re-run::
 
         pip install --editable .
 
-    every time the source code of a compiled extension is
-    changed (for instance when switching branches or pulling changes from upstream).
+    every time the source code of a compiled extension is changed (for
```

Comment:
I'd replace that, or maybe just mention, by `python setup.py build_ext -i` which really is what is needed.

Reply:
I'd rather not add another command, even if it's more strictly correct. I think introducing a single command is less scary for beginners.